src/program.ts

```diff
@@ -45,46 +42,11 @@ program
     .option('--user-data-dir <path>', 'Path to the user data directory')
     .option('--vision', 'Run server that uses screenshots (Aria snapshots are used by default)')
     .action(async options => {
-      let browserName: 'chromium' | 'firefox' | 'webkit';
-      let channel: string | undefined;
-      switch (options.browser) {
-        case 'chrome':
-        case 'chrome-beta':
-        case 'chrome-canary':
-        case 'chrome-dev':
-        case 'msedge':
-        case 'msedge-beta':
-        case 'msedge-canary':
-        case 'msedge-dev':
-          browserName = 'chromium';
-          channel = options.browser;
-          break;
-        case 'chromium':
-          browserName = 'chromium';
-          break;
-        case 'firefox':
-          browserName = 'firefox';
-          break;
-        case 'webkit':
-          browserName = 'webkit';
-          break;
-        default:
-          browserName = 'chromium';
-          channel = 'chrome';
-      }
-
-      const launchOptions: LaunchOptions = {
+      const serverList = new ServerList(() => createServer({
+        browser: options.browser,
+        userDataDir: options.userDataDir,
         headless: !!(options.headless ?? (os.platform() === 'linux' && !process.env.DISPLAY)),
```

Comment:
```suggestion
        headless: options.headless,
```
This logic already moved to index.ts